.github/workflows/daily-docker-build-lite-tmux.yml

```
@@ -0,0 +1,181 @@
+          "
+
+  cleanup-old-images:
+    needs: [merge, test-image]
+    runs-on: ubuntu-latest
+    if: success()
+    permissions:
+      packages: write
+
+    steps:
+      - name: Delete old daily images
+        uses: actions/delete-package-versions@v4
```

actionlint (1.7.11)

[error] the runner of "actions/delete-package-versions@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)